test/pizza-guy.spec.js

```diff
@@ -64,4 +79,27 @@ describe('pizza-guy', function() {
       expect(pizzaGuy.deliver([]).start()).toBe(undefined);
     });
   });
+
+  describe('physical files', function() {
+    it('should call onComplete when all the images where downloaded', async function() {
+      // TODO: Make this better when we know how to catch the completed images event
+      this.timeout(10000);
```

Comment:
I'm pretty sure we can get rid of this timeout using `process.nextTick`.
You can find more info about it here
- https://www.youtube.com/watch?v=-niA5XOlCWI
- https://howtonode.org/understanding-process-next-tick
- https://github.com/nodejs/node/blob/master/doc/topics/the-event-loop-timers-and-nexttick.md